CHANGELOG.md

```diff
@@ -27,5 +27,6 @@
 - Add `PartialEq` implementation for `LayerContentMetadata`.
 - Add `PartialEq` and `Eq` implementations for `LayerTypes`.
 - Add `LayerEnv::chainable_insert`
+- `LayerEnv` and `ModificationBehavior` now implement `Clone`.
```

Comment:
(I'm presuming `LayerEnvDelta` isn't listed since it's not public)

Reply:
Correct. :)